keps/sig-scheduling/2249-pod-affinity-namespace-selector/kep.yaml

```diff
@@ -0,0 +1,45 @@
+title: Namespace Selector for Pod Affinity
+kep-number: 2249
+authors:
+  - "@ahg-g"
+owning-sig: sig-scheduling
+participating-sigs:
+status: implementable
```

Comment:
should still be provisional if there are UNRESOLVED sections

Reply:
the plan is to get those resolved before merging.